libraries/xcversion.rb

```diff
@@ -3,37 +3,45 @@
 module MacOS
   module XCVersion
     class << self
-      def xcversion
-        '/opt/chef/embedded/bin/xcversion '.freeze
+      def xcversion_path
+        Chef::Util::PathHelper.join(Chef::Config.embedded_dir, 'bin', 'xcversion')
+      end
+
+      def xcversion(command)
+        [xcversion_path, command].join(' ')
```

Comment:
You could also write this function as:


```ruby
def xcversion(subcommand, *args)
  [xcversion, subcommand, *args].join ' '
end
```

This would allow us to specify each subcommand and subsequent arguments separately, rather than string interpolating all the arguments including the subcommand.